src/gui/order_widget.py

```
@@ -10,15 +10,24 @@
 )
 from PySide6.QtCore import Qt, Slot, QTimer
 
+from ..database.cache import SupabaseCache
```

Pylint (3.3.7)

[error] Attempted relative import beyond top-level package

(E0402)